WMF Framework/WMFExploreFeedContentController.h

```diff
@@ -22,6 +31,67 @@ extern const NSInteger WMFExploreFeedMaximumNumberOfDays;
 - (void)updateNearbyForce:(BOOL)force completion:(nullable dispatch_block_t)completion;
 - (void)updateBackgroundSourcesWithCompletion:(void (^_Nonnull)(UIBackgroundFetchResult))completionHandler;
 
+// Preferences
+
+/**
+ Toggles all customizable content groups on or off for a given siteURL.
+
+ @param siteURL A Wikipedia site url for which all customizable content groups will be visible or hidden in the feed.
+ @param isOn A flag that indicates whether all customizable content groups should be visible or hidden for a given siteURL in the feed.
+ @param updateFeed A flag that indicates whether feed should be updated after Explore feed preferences are updated.
+ */
+-(void)toggleContentForSiteURL:(nonnull NSURL *)siteURL isOn:(BOOL)isOn updateFeed:(BOOL)updateFeed;
+/**
+ Toggles a content group of given kind on or off for all preferred languages.
+
+ @param contentGroupKind The kind of the content group that is about to be toggled on or off in the feed.
+ @param isOn A flag indicating whether the group should be visible in the feed or not.
+ */
+- (void)toggleContentGroupOfKind:(WMFContentGroupKind)contentGroupKind isOn:(BOOL)isOn;
+/**
+ Toggles a content group of given kind on or off for a given siteURL.
+
+ @param contentGroupKind The kind of the content group that is about to be toggled on or off in the feed.
+ @param isOn A flag indicating whether the group should be visible in the feed or not.
+ @param siteURL A Wikipedia site url for which a content group of given kind will be visible or hidden in the feed.
+ */
+- (void)toggleContentGroupOfKind:(WMFContentGroupKind)contentGroupKind isOn:(BOOL)isOn forSiteURL:(nonnull NSURL *)siteURL;
+
+/**
+ Updates explore feed preferences with a given default tab type.
+
+ @param defaultTabType Preferred default tab type.
+ */
+- (void)changeDefaultTabTo:(WMFAppDefaultTabType)defaultTabType;
```

Comment:
This might make more sense as an extension on `UserDefaults` since the default tab of the app seems beyond the scope of feed content